libpod/image/image.go

```diff
@@ -431,5 +465,77 @@ func (i *Image) Size() (*uint64, error) {
 		}
 	}
 	return nil, errors.Errorf("unable to determine size")
+}
+
+// DriverData gets the driver data from the store on a layer
+func (i *Image) DriverData() (*inspect.Data, error) {
+	topLayer, err := i.Layer()
+	if err != nil {
+		return nil, err
+	}
+	return driver.GetDriverData(i.imageruntime.store, topLayer.ID)
```

Comment:
Can we make efforts to remove the Driver package, and move its files into here?

Reply:
Doesn't have to be now, but I would love to see `common` and `driver` disappear from libpod.

Reply:
sure, long term agree